src/main/java/service/impl/StringValidatorServiceImpl.java

```diff
@@ -0,0 +1,70 @@
+package service.impl;
+
+import java.util.List;
+import service.StringValidatorService;
+
+public class StringValidatorServiceImpl implements StringValidatorService {
+    private static final int TITLE_INDEX = 0;
+    private static final int TYPE = 0;
+    private static final int FRUIT = 1;
+    private static final int AMOUNT = 2;
+    private static final int VALID_LENGTH = 3;
+    private static final String TITLE_STRING = "type,fruit,quantity";
+    private static final String SEPARATOR = ",";
+    private static final String[] TYPE_OF_TRANSACTION = new String[] {"b", "s", "p", "r"};
+    private static final String[] TYPE_OF_FRUITS = new String[] {"banana", "apple"};
```

Comment:
we can have different fruits in our store
let's remove this and corresponding validation check